handsontable/src/shortcuts/manager.js

```diff
@@ -83,29 +82,35 @@ export const createShortcutManager = ({ ownerWindow, beforeKeyDown, afterKeyDown
    */
   const keyRecorder = useRecorder(ownerWindow, beforeKeyDown, afterKeyDown, (event, keys) => {
     const activeContext = getContext(getActiveContextName());
+    let isExecutionCancelled = false;
 
     if (activeContext.hasShortcut(keys)) {
       // Processing just actions being in stack at the moment of shortcut pressing (without respecting additions/removals performed dynamically).
-      const shortcuts = activeContext.getShortcuts(keys).slice();
+      const shortcuts = activeContext.getShortcuts(keys);
```

Comment:
Why do you remove it? I'm surprised the tests pass. The above comments show a purpose for `slice`.

Reply:
The [`getShortcuts`](https://github.com/handsontable/handsontable/blob/feature/issue-9369/handsontable/src/shortcuts/context.js#L178) method returns always a new copy of an array. The `slice` call is redundant here, I guess.